homeassistant/components/switchbot/sensor.py

```diff
@@ -34,6 +35,7 @@
     ),
     "wifi_rssi": SensorEntityDescription(
         key="wifi_rssi",
+        name="Wi-fi signal strength",
```

Comment:
Incorrect spelling of Wi-Fi

```suggestion
        name="Wi-Fi signal strength",
```